components/Footer.js

```diff
@@ -17,6 +18,16 @@ export default function Footer() {
           width="212"
         />
       </a>
+      <Link href="https://www.guidestar.org/profile/84-4323038" target="_blank">
+        <a>
```

Comment:
I believe this `<a>` tag is unnecessary and actually gets removed entirely during the rendering process.

Reply:
Hey I've put both `vercel` and Guidestar seal in inline position. Also, I've removed redundant `<a>`  tag.